### PR TITLE
test pull request

### PR DESCRIPTION
**Pull request by zjnoel**
this is a test pull request
CHANGES  MADE:
added "apples" to readme

### CODE REVIEW

**Review by AndrewHouston — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,3 +13,6 @@ It's important that everyone involved in the pull request process carefully look
 **POSSIBLY USEFUL INFORMATION**
 1) Git documentation https://git-scm.com/docs <-- If you have never used GitHub or Git before, it is imperitive that you understand how it works so that your code can properly live on the repo. You can talk to Zach Noel if you are confused on the git process.
 2) You will need to have Git downloaded on your computer. Link is here: https://git-scm.com/download/win . I can provide install documentation later.
+
+
+apples
```
Comment (Hoffbot515) [inline README.md]: Why apples? Remove

**Review by Hoffbot515 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AndrewHouston) [inline README.md]: apples are tasty

**Review by AndrewHouston — DISMISSED** [review]

**Comment by ghost** [comment]
test

**Review by jonjojojon — CHANGES_REQUESTED** [review]
remove apples

**Review by ghost — APPROVED** [review]

**Review by Hoffbot515 — APPROVED** [review]

**Review by jonjojojon — APPROVED** [review]

**Review by AndrewHouston — APPROVED** [review]

**Pull request closed without merging**